korlibs-logger/src@js/korlibs/logger/Logger.js.kt

```diff
@@ -20,14 +19,17 @@ actual object DefaultLogOutput : Logger.Output {
     override fun output(logger: Logger, level: Logger.Level, msg: Any?) = Logger.ConsoleLogOutput.output(logger, level, msg)
 }
 
-external private val process: dynamic
-external private val Deno: dynamic
+private external val process: dynamic
+private external val Deno: dynamic
+
+private val isDenoJs: Boolean by lazy { js("(typeof Deno === 'object' && Deno.statSync)").unsafeCast<Boolean>() }
```

Comment:
Why do you duplicate platform code in logger? Why is it a problem to depend on platform?

Reply:
Because just for two lines, I don't see adding an extra dependency in this case.

Reply:
Hopefully in Kotlin 2 we will have the ability to differentiate runtimes integrated in the stdlib at some point.